languageserver/integration/resolvers_test.go

```diff
@@ -0,0 +1,105 @@
+/*
+ * Cadence - The resource-oriented smart contract programming language
+ *
+ * Copyright 2019-2022 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package integration
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/onflow/cadence/runtime/common"
+	"github.com/onflow/flow-go-sdk"
+
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_FileImport(t *testing.T) {
+	t.Parallel()
+	mockFS := afero.NewMemMapFs()
+	af := afero.Afero{Fs: mockFS}
+	_ = afero.WriteFile(mockFS, "./test.cdc", []byte(`hello test`), 0644)
+
+	resolver := resolvers{
+		loader: af,
+	}
+
+	t.Run("existing file", func(t *testing.T) {
+		resolved, err := resolver.fileImport("./test.cdc")
```

Comment:
Can run in parallel here and in other tests
```suggestion
		t.Parallel()
		resolved, err := resolver.fileImport("./test.cdc")
```

Reply:
I think so, good idea.

Reply:
Will make sure all run in parallel